x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -244,18 +244,23 @@ export const getManagementFilteredLinks = async (
   plugins: StartPlugins
 ): Promise<LinkItem> => {
   const fleetAuthz = plugins.fleet?.authz;
-  const { endpointRbacEnabled, endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
+  const { endpointRbacV1Enabled } = ExperimentalFeaturesService.get();
   const endpointPermissions = calculatePermissionsFromCapabilities(core.application.capabilities);
```

Comment:
can you rename this variable for clarity?  I took a look at this function `calculatePermissionsFromCapabilities()` and it does not return endpoint Authz - it returns two properties that indiate if the user has access to Security Solution (the entire app).

Reply:
4e7b6907a97cfc72774f257bff23148aadf223e1